sklearn/feature_extraction/tests/test_text.py

```diff
@@ -324,6 +324,23 @@ def test_tf_idf_smoothing():
     assert_true((tfidf >= 0).all())
 
 
+def test_tfidf_partial_fit():
+    X = [[1, 1, 1],
+         [1, 1, 0],
+         [1, 0, 0]]
+
+    tr_full = TfidfTransformer(smooth_idf=True, norm='l2')
+    tfidf_full = tr_full.fit_transform(X).toarray()
+
+    tr_partial = TfidfTransformer(smooth_idf=True, norm='l2')
+    tr_partial.fit([[1, 1, 1]])
```

Comment:
I was wondering what the semantics would be if the 'partial_fit cannot change the number of features' (see the issue discussion).
Ok, will re-do